tests/load_tests/main.yml

```diff
@@ -0,0 +1,21 @@
+config:
```

Comment:
Is it possible to check the HTTP status of the request? This will be important when we are redirecting. You should get a 200 now and a 302 when the redirect is in place.

Reply:
Yes, it's possible, via Headers. As we discussed, I not only need to check for the 302 status but also for the destination (ensure that it is to a rex url/domain) as we currently already redirect.